keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk

```
@@ -0,0 +1,5 @@
+RGBLIGHT_ENABLE = no
+MOUSEKEY_ENABLE = no
+DEBOUNCE_TYPE = sym_eager_pk
+VIA_ENABLE = yes
+LT0_ENABLE = yes
```

```suggestion
LTO_ENABLE = yes
```
this still needs to be resolved.